File: VeamIt-project/iOS/veam00000000/veam00000000/TemplateMixed.m
//
//  TemplateMixed.m
//  veam00000000
//
//  Created by veam on 6/13/14.
//  Copyright (c) 2014 veam. All rights reserved.
//

#import "TemplateMixed.h"

@implementation TemplateMixed

@synthesize templateMixedId ;
@synthesize title ;

- (id)initWithAttributes:(NSDictionary *)attributeDict
{
    self = [super init] ;
    
    [self setTemplateMixedId:[attributeDict objectForKey:@"id"]] ;
    [self setTitle:[attributeDict objectForKey:@"t"]] ;
    
    return self ;
}


- (void)handlePostResult:(NSArray *)results
{
    //NSLog(@"%@::handlePostResult",NSStringFromClass([self class])) ;
    if([results count] >= 1){
        //NSLog(@"count >= 1") ;
        NSString *result = [results objectAtIndex:0] ;
        if([result isEqualToString:@"OK"]){
        }
    }
}


@end

File: VeamIt-project/Android/veam31000287/veam31000287/src/main/java/co/veam/veam31000287/TemplateMixed.java
package co.veam.veam31000287;

import org.xml.sax.Attributes;

/**
 * Created by veam on 10/27/16.
 */
public class TemplateMixed {
    private String templateMixedId ;
    private String title ;

    public TemplateMixed() {

    }

    public String getTemplateMixedId() {
        return templateMixedId;
    }

    public void setTemplateMixedId(String templateMixedId) {
        this.templateMixedId = templateMixedId;
    }

    public String getTitle() {
        return title;
    }

    public void setTitle(String title) {
        this.title = title;
    }

    public TemplateMixed(Attributes attributes) {
        this.setTemplateMixedId(attributes.getValue("id")) ;
        this.setTitle(attributes.getValue("t")) ;
    }
}
